src/pages/pass-finalisation/cat-c/_tests_/pass-finalisation.cat-c.page.spec.ts

```diff
@@ -201,5 +202,27 @@ describe('PassFinalisationCatCPage', () => {
       });
 
     });
+
+    describe('showCode78', () => {
+      it('should return true when the test is a category C', () => {
+        component.testCategory = TestCategory.C;
+        expect(component.showCode78()).toBeTruthy();
```

Comment:
tobeTruthy() doesn't do what you think it does we should be using toEqual(true) or toBe(true) - This article should explain why - https://vincenttunru.com/toBeTruthy-vs-toBe-true/